cmd/okctl/apply_application.go

```diff
@@ -46,41 +49,17 @@ func buildApplyApplicationCommand(o *okctl.Okctl) *cobra.Command {
 				return err
 			}
 
-			scaffoldOpts.ID = &api.ID{
+			opts.ClusterID = api.ID{
 				Region:       o.Declaration.Metadata.Region,
 				AWSAccountID: o.Declaration.Metadata.AccountID,
 				ClusterName:  o.Declaration.Metadata.Name,
 			}
 
-			scaffoldOpts.Application, err = commands.InferApplicationFromStdinOrFile(o.In, o.FileSystem, opts.File)
+			opts.Application, err = commands.InferApplicationFromStdinOrFile(o.In, o.FileSystem, opts.File)
 			if err != nil {
 				return fmt.Errorf("inferring application from stdin or file: %w", err)
 			}
 
-			scaffoldOpts.OutputDir, err = o.GetRepoOutputDir()
```

Comment:
Should we comment in o.GetRepoOutputDir that it is deprecated, read state from cluster declaration instead?